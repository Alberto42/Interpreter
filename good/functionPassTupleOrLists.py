def f(x) {
    print(x)
}

f((1,True,"str"))
f([1,2,3])