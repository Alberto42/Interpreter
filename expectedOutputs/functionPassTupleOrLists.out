
(1,True,str)
[1,2,3]

